docs/content/querying/select-query.md

```diff
@@ -24,7 +24,15 @@ title: "Select Queries"
 
 # Select Queries
 
-Select queries return raw Apache Druid (incubating) rows and support pagination.
+<div class="note caution">
+We encourage you to use the [Scan query](../querying/scan-query.html) query type rather than Select whenever possible.
```

Comment:
[Scan query] query type -> [Scan query] type?

Reply:
Fixed this, thanks